doc/src/content/xdocs/spec.xml

```diff
@@ -546,7 +546,7 @@
 
           <section id="union_encoding">
             <title>Unions</title>
-            <p>A union is encoded by first writing a <code>long</code>
+            <p>A union is encoded by first writing an <code>int</code>
```

Comment:
Until the needed consideration for c/++ are done, I think we should include a parenthetical for that case. Something like _(this can theoretically be a long, and is allowed as such in C and C++, but we don't anticipate unions with 215M members)_.

Reply:
@kojiromike It's very unlikely to have unions with such members. We should open a ticket to track as [you suggested](https://lists.apache.org/thread.html/rac728c7e4fd8052778e560b997e473745453eec3c3a1826b3cc3607c%40%3Cuser.avro.apache.org%3E).

Reply:
@anhldbk Agreed. I'm also asking you to add a similar note right here in the docs, either as a footnote with an asterisk, or with parentheses. Sure, we'll never see such a union in real life, but we should still document the state of the implementation(s) as they are. That is, for C and C++, practically an `int`, but theoretically a `long`.